apps/web/cypress/tests/integrations-list-page.spec.ts

```diff
@@ -32,23 +32,56 @@ describe('Integrations List Page', function () {
     }: {
       name: string;
       isFree?: boolean;
-      key: string;
+      key: string | null;
       provider: string;
       channel: string;
-      environment: string;
+      environment: string | null;
```

Comment:
What's the reason of not making it optional?
```suggestion
      environment?: string;
```